### PR TITLE
Cashella event

### PR DESCRIPTION
**Pull request by dr-ishmael**
Everything for Cashella. Definitely more cash upgrades to come, possibly more angel upgrades.

### CODE REVIEW

**Comment by ricardopetrere** [comment]
Damn, beat me to it!! I was doing the exact same thing! hahahaha
Issue: All the exponentials must have a "+" before the exponent (ex: 202.02e+27)
But here, have a few cash upgrades I saw :)
`$scope.cashella.cashUpgrades = [[22345, [0, 2], false],[127890, [2, 2], false],[243456, [4, 2], false],[1569012, [6, 2], false],[2464567, [8, 2], false],[17801234, [10, 2], false],[24691356, [12, 2], false],[180246912, [14, 2], false],[246913578, [16, 2], false],[2.469e+9, [18, 3], false],[49.362e+9, [0, 5], false],[738.243e+9, [2, 5], false],[9.624e+12, [4, 5], false],[102.03e+12, [6, 5], false],[1.224e+15, [8, 5], false],[14.284e+15, [10, 5], false],[163.248e+15, [12, 5], false],[1.836e+18, [14, 5], false],[2.04e+18, [16, 5], false],[22.446e+18, [18, 7], false],[222.222e+18, [0, 9], false],[4.444e+21, [2, 9], false],[66.666e+21, [4, 9], false],[888.888e+21, [6, 9], false],[5.555e+24, [8, 9], false],[13.333e+24, [10, 9], false],[155.555e+24, [12, 9], false],[1.777e+27, [14, 9], false],[20e+27, [16, 9], false],[202.02e+27, [18, 11], false],[6.283e+30, [0, 13], false],[100.576e+30, [2, 13], false],[461.563e+30, [4, 13], false]`

**Comment by dr-ishmael** [comment]
The plus appears to be optional, because it works just fine for me without it. https://dr-ishmael.github.io/Slimmmo.github.io/

I already have the cash upgrades to the same point you do, and a couple more.

**Comment by ricardopetrere** [comment]
Yeah, so I saw... I got confused by the 'e'...
My bad

**Comment by 61352151511** [comment]
Merge conflicts created by the last PR, please fix and then this will be merged

**Comment by ricardopetrere** [comment]
At line 1366, shouldn't there be "49.362e9" instead of "46.362e9"?

**Comment by dr-ishmael** [comment]
Goddammit. Is the conflict on both files or just one of them?

File: index.html [hunk 1]
```diff
@@ -182,7 +182,18 @@
...
               <label class="col-xs-6"><input type="number" class="form-control" min="0" data-ng-model="ref.bonusMultiplier">x multiplier</label>
-              <label class="col-xs-6">Total Angel Investor % = {{ref.angelEffectiveness | number:0}}%</label>
+              <label class="col-xs-6">Total Angel Investor % = {{ref.angelEffectiveness | number}}%</label>
+            </uib-accordion-group>
+            <uib-accordion-group is-open="accOpen[6]">
+              <uib-accordion-heading>
+                Super Badges<span class="pull-right fa" ng-class="{'fa-chevron-down': accOpen[6], 'fa-chevron-right': !accOpen[6]}"></span>
+              </uib-accordion-heading>
+              <div class="multicolumn">
+              <div class="multicolumn" data-ng-show="!isEvent()">
```
Comment (ricardopetrere) [inline index.html]: This line should be deleted. The above (191) is the correct, because Super badges can be used on events

**Comment by dr-ishmael** [comment]
I don't know what's going on here, I can't figure out where the remaining conflicts are. I'm just going to scrap this fork, make a new one, and recommit the data.

**Pull request closed without merging**